src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-wrapper">
+      <form>
+        <div
+          class="search-form search-form--big"
+          data-qa="big"
+        >
+          <input
+            type="text"
+            placeholder="Try 'Los Angeles'"
+            data-qa="keypress"
+          />
+        </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+        <div
+          class="search-form search-form--small"
+          data-qa="small"
+        >
+          <input
+            type="text"
+            placeholder="Try 'Los Angeles'"
+          />
```

Comment:
Similar to the previous input, consider adding a `name` attribute to this input element for better form data handling and accessibility.